sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1854 @@
+and fixed features.  A rationale is provided for each proposed modification to
+Stacks.
+
+# Specification
+
+Applying these upgrades to the Stacks blockchain requires a
+consensus-breaking network upgrade, in this case, a hard fork. Like
+other such changes, this will require a new Stacks epoch. In this SIP,
+we will refer to this new epoch as Stacks 2.1.
+
+At the onset of Stacks 2.1, the Clarity VM will begin to support
+"Clarity Version 2". This version will include support for the new
+native methods proposed in this SIP (and therefore include new
+*keywords* which cannot be used for method names or variable names).
+New contracts launched in Stacks 2.1 will _default_ to Clarity 2, but
+contract authors will be able to use a new contract-publish transaction type to
+indicate if a contract should specifically launch using
+Clarity 1 or Clarity 2.
+
+Additionally, a new `pox-2` contract will be
+published by the boot address. The `stacks-node` will use the new
+`pox-2` contract for determining PoX reward sets and governing PoX
+locks.  Similarly, new rules for block validation take effect at the onset of Stacks
+2.1.
+
+## Stacking
+
+This section of the SIP specifies the new and changed behaviors of the
+PoX-2 contract and also provides new use cases that demonstrate how
+the new PoX-2 contract could be used.
+
+### Overview
+
+The new PoX contract operates exclusively with PoX state that was
+created in the new contract. It will not "import" state from the
+original PoX contract. In order to allow for this, a particular reward
+cycle `N` is chosen as the "Last PoX-1" reward cycle, and then the
+"First PoX-2" reward cycle is the subsequent cycle (`N+1`).  In-between these
+two cycles, there exists a burnchain block height `v1_unlock_height` at which
+all of the STX tokens in PoX-1 will unlock, so that they can be locked up again
+in PoX-2 before cycle `N+1` starts.
+
+This defines three periods of PoX operation:
+
+Period 1 | Period 2a | Period 2b | Period 3
+-- | -- | -- | --
+2.0 Consensus Rules in Effect | 2.1 Consensus Rules enacted, PoX-2 exists, but PoX-1 is still active | PoX-1 deactivates, all tokens locked in the original PoX contract unlock, but first PoX-2 reward cycle has not begun | First PoX-2 reward cycle has begun
+_This is the period of time before the 2.1 fork._  |  _This is the period of time after the 2.1 fork, and before `v1_unlock_height` (inclusive)_ | _This is the period of time between `v1_unlock_height + 1` and the start of cycle (N+1)_ |  _This is the start of cycle (N+1), and all cycles afterward.  The original PoX contract state will no longer have any impact on reward sets, account lock status, etc._
+
+- Every account that is locked by PoX-1 for cycle `N`
+  and beyond is unlocked at the end of period 2a, once the burnchain block
+height `v1_unlock_height` passes.
+- Accounts locked in PoX-2 will remain locked.  It will be possible to do this
+  with unlocked STX at the start of period 2a.
```

Comment:
>  Accounts locked in PoX-2 will remain locked.  It will be possible to do this
   with unlocked STX at the start of period 2a.

This still trips me up a bit. Could it be rephrased? What action is meant by "do this"?